sass/themes/okc/_theme.scss

```diff
@@ -23,3 +23,28 @@ nav.top-bar {
 	border-bottom-style: inset;
 	border-bottom-color: $primary-color;
 }
+
+/* When redefining styles, be more specific as original style
+   definition, so that it gets applied. And mind, that default styles
+   might define additional modifications for classes that will get
+   overriden ... */
+.itinerary-summary-row {
+	.itinerary-legs {
+		.leg {
+			.bar-container {
+				div.bar {
+					border-color: currentColor;
+					background-color: white;
+					border: 1px solid;
+				}
+				.route-number {
+					.vehicle-number-container-v {
+						.vehicle-number {
+							color: black;
```

Comment:
In `sass/themes/default/_theme.scss`, we have `$black: #333`. Shall we use that?

Reply:
Yes, will switch this, though we might darken it later a bit to improve contrast.